manifests/install.pp

```diff
@@ -17,17 +17,18 @@
   $package_options        = undef,
 ) {
   include tomcat
+  $_catalina_home = pick($catalina_home, $::tomcat::catalina_home, $name)
```

Comment:
I agree in that the catalina_home set by the main class should be the default. However, in doing so.. we will have to do a major version bump. I can keep this PR around with an appropriate milestone label.